metabase/metabase
# Retain field and card values when switching to custom list

Epic https://github.com/metabase/metabase/issues/26898

How to verify:
- New -> Dashboard -> Filter -> Text or Category -> Is
- Dropdown list -> Edit
- From another model or question -> Pick a question that has string columns -> Pick a string column
- You should see the values for this field in the selected question
- Switch to Custom list. The values should be retained, and it should be possible to modify and save them.

<img width="614" alt="Screenshot 2023-02-07 at 13 25 29" src="https://user-images.githubusercontent.com/8542534/217233585-ee9a6808-0fd5-420b-b9d2-9d2f3cf2ab93.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/28117)
